src/databricks/labs/ucx/contexts/application.py

```diff
@@ -281,7 +282,10 @@ def aws_acl(self):
     def principal_locations(self):
         eligible_locations = {}
         if self.is_azure:
-            eligible_locations = self.azure_acl.get_eligible_locations_principals()
+            try:
+                eligible_locations = self.azure_acl.get_eligible_locations_principals()
+            except ResourceDoesNotExist:
```

Comment:
can you cast try-except wider, so that the aws case is also covered?

Reply:
done